javascript/builtins/Atomics.json

```diff
@@ -102,7 +102,8 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": false
+              "version_added": "8.0",
```

Comment:
This whole file needs reverting,

atomics came in as part of shared array buffers which allowed syncing an array buffer between threads with atomic operations.

Unfortunately shortly after it landed the spectre attack was revealed.

The tight timing of atomics was a vector for this attack so the API was needed to be reimplemented.

As a result never actually landed in Samsung Internet.